### PR TITLE
Added point cloud workshop steps

### PR DESCRIPTION
**Pull request by ninok**

### CODE REVIEW

File: harp.gl/grace-hopper-2020/point-clouds/final/.eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{
```
Comment (philjhere) [inline harp.gl/grace-hopper-2020/point-clouds/final/.eslintrc.json:1]: Hi Nino, I would suggest removing files like the .gitignore and this eslint file. from the repo.  Granted this is your tutorial and I'd leave it to you for the final call. ~Phil

File: harp.gl/grace-hopper-2020/point-clouds/final/index.html [hunk 2]
```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
```
Comment (philjhere) [inline harp.gl/grace-hopper-2020/point-clouds/final/index.html:1]: Should probably add the License

File: harp.gl/grace-hopper-2020/point-clouds/final/resources/csv-to-json.py [hunk 3]
```diff
@@ -0,0 +1,33 @@
+#!/bin/python
```
Comment (philjhere) [inline harp.gl/grace-hopper-2020/point-clouds/final/resources/csv-to-json.py:1]: I'm not sure if this came from somewhere else, but if so we need to identify who the author is or give the License attribution at the top of the file.

Comment (ninok) [reply harp.gl/grace-hopper-2020/point-clouds/final/resources/csv-to-json.py:1]: Okay, will add license header to all the files.

File: harp.gl/grace-hopper-2020/point-clouds/final/.eslintrc.json (hunk 1 above)
Comment (ninok) [reply harp.gl/grace-hopper-2020/point-clouds/final/.eslintrc.json:1]: yeah, I just added them b/c the auto formatting was driving me nuts. Will remove them.

**Pull request merged**